content/en/llm_observability/trace_proxy_services.md

```diff
@@ -0,0 +1,185 @@
+
+LLMObs.enable(service="chat-proxy")
+
+@app.route('/chat')
+def chat():
+    with LLMObs.workflow(name="chat-proxy-entrypoint"):
+        # proxy-specific logic, including guardrails, sensitive data scans, and the LLM call
+```
+
+{{% /tab %}}
+{{% tab "Node.js" %}}
+
+```javascript
+// proxy.js
+const tracer = require('dd-trace').init({
+  llmobs: true,
+  service: "chat-proxy"
+});
+const llmobs = tracer.llmobs;
+
+app.post('/chat', async (req, res) => {
+  await llmobs.trace({ name: 'chat-proxy-entrypoint', kind: 'workflow' }, async () => {
+    // proxy-specific logic, including guardrails, sensitive data scans, and the LLM call
+    res.send("Hello, world!");
+  });
+});
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+All requests to the proxy service can now be viewed as top-level spans within the LLM trace view:
+
+1. In the LLM trace view, view `All Applications` from the top-left dropdown.
+2. Switch to the `All Spans` view in the top-right dropdown.
+3. Filter the list by the `service` tag and the workflow name.
+
+{{< img src="llm_observability/all-spans-with-service-and-span-name.png" alt="View all spans from all ML applications with the service and workflow name tags" style="width:100%;" >}}
+
+The workflow span name can also be filtered by a facet on the left hand side of the trace view:
+
+{{< img src="llm_observability/span-name-facet-for-proxy-service-monitoring.png" alt="Select the workflow span name from the facet on the left hand side of the trace view" style="width:50%;" >}}
+
+### All LLM calls made within the proxy or gateway service
+
+To instead monitor only the LLM calls made within a proxy or gateway service, filter by `llm` spans in the trace view:
+
+{{< img src="llm_observability/all-spans-with-service-and-span-kind.png" alt="View all spans from all ML applications with the service tags and the LLM span kind" style="width:100%;" >}}
+
+This can also be done by filtering the span kind facet on the left hand side of the trace view:
+
+{{< img src="llm_observability/span-kind-facet-for-proxy-service-monitoring.png" alt="Select the LLM span kind facet from the left hand side of the trace view" style="width:50%;" >}}
+
+### Filtering by a specific ML application and observing patterns and trends
+
+Both processes described in the filtering by [top-level calls to the proxy service](#all-requests-to-the-proxy-or-gateway-service) and [LLM calls made within the proxy or gateway service](#all-llm-calls-made-within-the-proxy-or-gateway-service) can also be applied to a specific ML application to view its interaction with the proxy or gateway service.
+
+1. In the top-left dropdown, select the ML application of interest.
+2. To see all traces for the ML application, switch from the `All Spans` view to the `Traces` view in the top-right dropdown.
+3. To see a timeseries of traces for the ML application, switch from a `List` view to a `Timeseries` view in the Traces view while maintaining the `All Span` filter in the top-right dropdown.
```

Comment:
Since this is a numbered list, there's an assumption that a user is going through each of these in order. So I would assume they are in the Traces view from step 2.

```suggestion
3. To see a timeseries of traces for the ML application, switch back to the **All Spans** filter in the top-right dropdown and next to "Visualize as", select **Timeseries**.
```